low-level/sensors/impl/centos_7/service_monitor.py

```diff
@@ -446,7 +446,7 @@ def check_nonactive_services(self):
             if self.services[service].is_nonactive_for_threshold_time():
                 self.services[service].new_service_state(FailedState)
                 self.raise_alert(self.get_alert(self.services[service],
-                                                FailedAlert))
+                                                InactiveAlert))
```

Comment:
I believe the bug is about updating description, to reflect that SSPL waited for configured time (180 seconds) before raising the alerts !? Why change in service alert type here

Reply:
FailedAlert, InactiveAlert its an namedtuple, 
Here we have to pass InactiveALert in this tuple we have defined the description as - eg. "statsd.service in inactive state for more than 180 seconds."

FailedAlert = Alert("fault", "{} in {} state.", "{} service is unavailable.",
                    "Try to restart the service")
InactiveAlert = Alert("fault", "{} in {} state for more than {} seconds.",
                      "{} service is unavailable.",
                      "Try to restart the service")
ResolvedAlert = Alert("fault_resolved", "{} in {} state.",
                      "{} service is available now.",
                      "No action required")

Reply:
>Stable R2: #251: Fault description for activating, deactivating alerts doesn't include the details related to time for which it has waited before reporting the alert.

Before raising fault-resolved alert, logic should wait n see if service stays recovered for sometime and add in description about this time wait too. 

>ResolvedAlert = Alert("fault_resolved", "{} in {} state **from last {} seconds.**",
"{} service is available now.",
"No action required")


Am i missing something ?

Reply:
No @cdeshmukh , the requirement from the bug is , if the service remains in say in activating state for more that 180 secs, then we used to give fault description as `"statsd.service in activating state for more than 180 seconds"`. But with the latest changes by Sandeep, by mistake, we used the `FaultAlert` msg template instead of `InactiveAlert` template, so it gives `"statsd.service in activating state."` as a description.
So we just needed to use that `InactiveAlert` description to correct it.

```Before raising fault-resolved alert, logic should wait n see if service stays recovered for sometime and add in description about this time wait too.```
This is the new thing that we can think of addding.

Reply:
With current implementation:
We wait fo threshhold time(180 sec) only if service is in intermediate state i.e - inactive, deactivating, activating.
As soon we detect service state changed from any intermediate state to active or failed to active, we raised the fault_resolved alert.
Should I add wait for fault_resolved case also??
@cdeshmukh

Reply:
>But with the latest changes by Sandeep, by mistake, we used the FaultAlert msg template instead of InactiveAlert template, so it gives "statsd.service in activating state." as a description.
So we just needed to use that InactiveAlert description to correct it.

@sumedhak27 of then lets correct that, but bug is about fault-resolved mainly

>Should I add wait for fault_resolved case also??

@shriya-deshmukh  yes, thats what i meant. to avoid raising too many alerts if state transition happens quite often. rare but still possible

Reply:
Done, added wait/threshold time for fault-resolved alert and updated description as.
ie. 
ResolvedAlert = Alert("fault_resolved", **"{} in {} state from last {} seconds."**,
"{} service is available now.",
"No action required")